.github/workflows/CI.yml

```diff
@@ -98,7 +98,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: ECP-CoPA/Cabana
-          ref: 0.6.1
+          ref: master
```

Comment:
We try to keep the CI on a tagged version and Nightly on master/develop, but if we need anything newer we can pick a hash

Reply:
Makes sense. I have rebased that change out, will switch to hash later if necessary.